controller.go

```diff
@@ -35,14 +38,22 @@ import (
 const (
 	defaultInvitationEmailTemplate = `Hello developer of great software, Kubernetes engineer or fellow human,
 
-A user of APPUiO Cloud has invited you to join them. Follow https://portal.dev/invitations/{{.Invitation.ObjectMeta.Name}}?token={{.Invitation.Status.Token}} to accept this invitation.
+A user of APPUiO Cloud has invited you to join them. Follow https://portal.dev/invitations/{{.Object.ObjectMeta.Name}}?token={{.Object.Status.Token}} to accept this invitation.
 
 APPUiO Cloud is a shared Kubernetes offering based on OpenShift provided by https://vshn.ch.
 
 Unsure what to do next? Accept this invitation using the link above, login to one of the zones listed at https://portal.appuio.cloud/zones, deploy your application. A getting started guide on how to do so, is available at https://docs.appuio.cloud/user/tutorials/getting-started.html. To learn more about APPUiO Cloud in general, please visit https://appuio.cloud. 
 
 If you have any problems or questions, please email us at support@appuio.ch.
 
+All the best
+Your APPUiO Cloud Team`
+	defaultBillingEntityEmailTemplate = `Good time of day!
+
+A user of APPUiO Cloud has updated billing entity {{.Object.ObjectMeta.Name}} ({{.Object.Spec.Name}}).
+
+See https://erp.vshn.net/web#id={{ trimPrefix "be-" .Object.ObjectMeta.Name }}&view_type=form&model=res.partner&menu_id=74&action=60 for details.
```

Comment:
Probably fine to default to erp.vshn.net here, since the invitation template isn't consistently using fake default URLs anymore either.

@corvus-ch what do you think?